src/main/java/io/shelang/aghab/service/link/LinksServiceImpl.java

```diff
@@ -38,7 +52,7 @@ public LinksDTO getById(Long id) {
 
   @Override
   public LinksDTO getByHash(String hash) {
-    Links links = linksRepository.findByHash(hash).orElseThrow(NotFoundException::new);
+    var links = linksRepository.findByHash(hash).orElseThrow(NotFoundException::new);
```

Comment:
JDK 10, an implementation of Java Standard Edition 10, was released on March 20, 2018.